cdhweb/pages/models.py

```diff
@@ -291,6 +292,15 @@ def get_context(self, request):
         # add up to 3 upcoming, published events
         context["events"] = Event.objects.live().upcoming()[:3]
 
+        # add "featured pages" with special section: currently about/consult,
+        # don't add them to context if not published
+        # NOTE effectively hardcoding by slug for now; could generalize later
+        context["about"] = ContentPage.objects.live().filter(slug="about").first()
+        context["consult"] = ContentPage.objects.live().filter(slug="consult").first()
```

Comment:
When I'm setting multiple things in a dictionary in a row, I prefer to use `update()`

Reply:
It seems ok to me hardcode by slug for now, but definitely worth noting